torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9366,7 +9366,10 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
     OpInfo('resize_',
            op=lambda x, shape: x.clone().resize_(shape),
            method_variant=None,
-           inplace_variant=None,
```

Comment:
what does it mean to test in-place variant for an in-place op?

Reply:
The way inplace variant is obtained is by adding an `_` to the op name, so I am a bit surprised that it's doing anything sensible here at all. Also, that shows (as do the assumptions made my many tests) that we assume that the op input to `OpInfo` is an out-of-place operation. The correct thing here would probably be to have a way to specify if the input op is in-place and doesn't have an out-of-place version.

cc. @mruberry

Reply:
We could do that but it's easier to model the OpInfo like we do here where we create a functional variant for it using a lambda. I also noticed this didn't have an inplace variant, which is odd, and this is a good coverage fix.

The automatic lookup of function, method, and inplace variants is convenient but not definitive, and in this case it will fail because "resize__" is not a method, so the inplace variant needs to be specified explicitly.